website/docs/reference/warehouse-profiles/iomete-profile.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Apache iomete Profile"
```

Comment:
nit: is iomete an official Apache project? I know Iceberg and Spark are, but this combo isn't?
```suggestion
title: "iomete Profile"
```